app/assets/javascripts/discourse/app/lib/codeblock-buttons.js

```diff
@@ -119,11 +119,7 @@ export default class CodeblockButtons {
         }px`;
       }
 
-      if (
-        this.showFullscreen &&
-        !Mobile.isMobileDevice &&
-        codeBlock.scrollWidth > codeBlock.clientWidth
```

Comment:
Just seemed like a pointless restriction in the end; why not just show the fullscreen button all the time? (I added this originally)

Reply:
I suppose it's a bit weird to show the fullscreen button when the modal is already at max size on small screens?

Reply:
Ah yeah I didn't realise it doubled up with that button...will think of something